src/gkApi.ts

```diff
@@ -7,13 +7,22 @@ const gkDotDevUrl = 'https://gitkraken.dev';
 const accessTokenCookieName = 'accessToken';
 
 const getAccessToken = async () => {
-	// Attempt to get the access token cookie from GitKraken.dev
-	const cookie = await cookies.get({
-		url: gkDotDevUrl,
-		name: accessTokenCookieName,
-	});
+	try {
+		const cookie = await cookies.get({
+			url: gkDotDevUrl,
+			name: accessTokenCookieName,
+		});
 
-	return cookie?.value;
+		return cookie?.value;
+	} catch (e) {
+		if ((e as Error)?.message.includes('No host permissions for cookies at url')) {
+			// ignore as we are waiting for required permissions
+		} else {
+			// otherwise log error and continue as if logged out
+			console.error(e);
+		}
+	}
+	return undefined;
```

Comment:
instead of try/catch, would it be better to check to see if we have the permission first?

Reply:
yeah i felt bad cause it adds an extra await but it's cleaner. will do :+1: